docs/Advanced/Tasks Api.md

```diff
@@ -103,3 +112,20 @@ Use these steps to make the following options appear (tested in QuickAdd 0.12.0)
 
 Screenshot of QuickAdd capture settings (example)
 ![Screenshot - Edit the QuickAdd Capture Configuration](../../images/api-create-taskline-modal-quickadd-capture-example.png)
+
+## `executeToggleTaskDoneCommand: (line: string, path: string) => string;`
+
+> [!released]
+This method was introduced in Tasks X.Y.Z.
+
+Executes the 'Tasks: Toggle task done' command on the supplied line string. It toggles and updates a task line according to a user's preferences, accounting for recurrance rules and completed status. It returns a string representing the toggled task.
+
+```typescript
+const tasksApi = this.app.plugins.plugins['obsidian-tasks-plugin'].apiV1;
+const sourceFile: TFile = file;
+const taskLine = '- [ ] This is a task 📅 2024-04-24';
+
+const result = tasksApi.toggleLine(taskLine, sourceFile.path);
```

Comment:
Shouldn't we call the `executeToggleTaskDoneCommand()` here?

Reply:
Oops! Fixed